src/tests/test_mceliece.cpp

```diff
@@ -148,17 +151,20 @@ class McEliece_Tests : public Test
          struct keygen_params { size_t code_length, t_min, t_max; };
 
          const keygen_params param_sets[] = { { 256, 5, 15 },
-                                              { 512, 5, 33 },
-                                              { 1024, 15, 35 },
-                                              { 2048, 33, 50 },
-                                              { 6624, 110, 115 } };
+               { 512, 5, 33 },
```

Comment:
Add extra newline so 256, 5, 15 in first line lines up with the rest of the values.